familyAccount.php

```diff
@@ -0,0 +1,403 @@
+<?php
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+
+    // Primary Parent/Guardian
+    $firstName = $_POST['first-name'];
+    $lastName = $_POST['last-name'];
+    $birthdate = $_POST['birthdate'];
+    $address = $_POST['address'];
+    $city = $_POST['city'];
+    $state = $_POST['state'];
+    $zip = $_POST['zip'];
+    $email = $_POST['email'];
+    $phone = $_POST['phone'];
+    $phoneType = $_POST['phone-type'];
+    $secondaryPhone = $_POST['secondary-phone'];
+    $secondaryPhoneType = $_POST['secondary-phone-type'];
+
+    // Secondary Parent/Guardian
+    $firstName2 = $_POST['first-name2'] ?? null;
+    $lastName2 = $_POST['last-name2'] ?? null;
+    $birthdate2 = $_POST['birthdate2'] ?? null;
+    $address2 = $_POST['address2'] ?? null;
+    $city2 = $_POST['city2'] ?? null;
+    $state2 = $_POST['state2'] ?? null;
+    $zip2 = $_POST['zip2'] ?? null;
+    $email2 = $_POST['email2'] ?? null;
+    $phone2 = $_POST['phone2'] ?? null;
+    $phoneType2 = $_POST['phone-type2'] ?? null;
+    $secondaryPhone2 = $_POST['secondary-phone2'] ?? null;
+    $secondaryPhoneType2 = $_POST['secondary-phone-type2'] ?? null;
+
+    // Children
+    $children = $_POST['children'] ?? [];
+
+    // Emergency Contact Fields
+    $econtactFirstName = $_POST['econtact-first-name'];
+    $econtactLastName = $_POST['econtact-last-name'];
+    $econtactPhone = $_POST['econtact-phone'];
+    $econtactRelation = $_POST['econtact-relation'];
+
+    // Login Credentials
+    $password = $_POST['password'];
+    $passwordReenter = $_POST['password-reenter'];
+
+    // Handle children forms
+    if (isset($_POST['children']) && is_array($_POST['children'])) {
+        foreach ($_POST['children'] as $index => $child) {
+            $childFirstName = $child['first-name'] ?? null;
+            $childLastName = $child['last-name'] ?? null;
+            $childBirthdate = $child['birthdate'] ?? null;
+            $childGender = $child['gender'] ?? null;
+            $childMedicalNotes = $child['child-medical-notes'] ?? null;
+            $childAdditionalNotes = $child['child-additional-notes'] ?? null;
+    
+            echo "Child $index: $childFirstName $childLastName, DOB: $childBirthdate, Gender: $childGender, Medical Notes: $childMedicalNotes, Additional Notes: $childAdditionalNotes <br>";
+        }
+    }
+}
+?>
+
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <?php require_once('universal.inc') ?>
+        <title>Stafford Junction | Create Family Account</title>
+    </head>
+    <body>
+        <?php require_once('header.php') ?>
```

Comment:
I think the code will break here since header.php checks if the $_SESSION['logged_in'] variable is set. Since the family hasn't logged in yet, I think this line of code will redirect the user to the login page